### ISSUE TITLE
Packer crashes with port must be specified error when communicator is set to none

### ISSUE THREAD

**Issue by mhlias** [mitchellh/packer#3054]
I tried building a Windows 2012R2 image in AWS using EBS provider and communicator none as I didn't want to do anything else than run what's in the userdata.

Packer:

```
Packer v0.8.6
```

My template:
```
{
  "variables": {
    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
  },
  "builders": [
    {
      "type": "amazon-ebs",
      "communicator": "none",
      "access_key": "{{user `aws_access_key`}}",
      "secret_key": "{{user `aws_secret_key`}}",
      "region": "eu-west-1",
      "associate_public_ip_address": true,
      "source_ami": "ami-1ffa5a6c",
      "instance_type": "c3.large",
      "user_data_file": "includes/windows-bootstrap.ps1",
      "ami_name": "windows-custom",
    }
  ]
}
```
Output:
```
amazon-ebs output will be in this color.

==> amazon-ebs: Prevalidating AMI Name...
==> amazon-ebs: Inspecting the source AMI...
==> amazon-ebs: Creating temporary keypair: packer 5693906d-6e0b-7bc2-618c-ada9c787e15b
==> amazon-ebs: Deleting temporary keypair...
Build 'amazon-ebs' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> amazon-ebs: unexpected EOF

==> Builds finished but no artifacts were created.
panic: port must be set to a non-zero value.
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 25 [running]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/packer/builder/amazon/common.(*StepSecurityGroup).Run(0xc8201cffc0, 0x1c44ce0, 0xc820225e90, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/builder/amazon/common/step_security_group.go:36 +0x370
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/multistep.(*BasicRunner).Run(0xc820286040, 0x1c44ce0, 0xc820225e90)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/multistep/basic_runner.go:70 +0x330
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/packer/builder/amazon/ebs.(*Builder).Run(0xc8201d0000, 0x1c448c0, 0xc82025c8c0, 0x1c44908, 0xc82025ac80, 0x1c44938, 0xc8201f80a0, 0x0, 0x0, 0x0, ...)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/builder/amazon/ebs/builder.go:168 +0x15f7
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/packer/packer/rpc.(*BuilderServer).Run(0xc820176620, 0x1, 0xc820172540, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/packer/rpc/builder.go:93 +0x3b9
2016/01/11 11:22:23 packer-builder-amazon-ebs: reflect.Value.call(0x607560, 0x6b3320, 0x13, 0x772338, 0x4, 0xc820293ed8, 0x3, 0x3, 0x0, 0x0, ...)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/reflect/value.go:432 +0x120a
2016/01/11 11:22:23 packer-builder-amazon-ebs: reflect.Value.Call(0x607560, 0x6b3320, 0x13, 0xc820293ed8, 0x3, 0x3, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/reflect/value.go:300 +0xb1
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/rpc.(*service).call(0xc820178440, 0xc820178400, 0xc8201724b0, 0xc82017a580, 0xc8201f0360, 0x526ca0, 0xc820172530, 0xca, 0x50a700, 0xc820172540, ...)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/server.go:383 +0x1c1
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by net/rpc.(*Server).ServeCodec
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/server.go:477 +0x4ac
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 1 [select]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Stream).Read(0xc8201c2000, 0xc8201ed000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/stream.go:125 +0x3ae
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).fill(0xc82018a6c0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:97 +0x1e9
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).ReadByte(0xc82018a6c0, 0x6aa8e0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:229 +0x7a
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc820170780, 0x82)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/decode.go:90 +0x42
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNext(0xc820176720)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/msgpack.go:540 +0x41
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc82018a720, 0x502c60, 0xc820176740)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x70
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc82018a720, 0x502c60, 0xc820176740, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x74
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc820178740, 0x502c60, 0xc820176740, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd2
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadRequestHeader(0xc820178740, 0xc820176740, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/rpc.go:127 +0x54
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/rpc.(*Server).readRequestHeader(0xc820178400, 0x19a16a0, 0xc820178740, 0x0, 0x0, 0xc820176740, 0xc8200e3c00, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/server.go:576 +0x90
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/rpc.(*Server).readRequest(0xc820178400, 0x19a16a0, 0xc820178740, 0xc820178400, 0xc8201724b0, 0xc82017a580, 0x0, 0x0, 0x0, 0x0, ...)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/server.go:543 +0x8b
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/rpc.(*Server).ServeCodec(0xc820178400, 0x19a16a0, 0xc820178740)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/server.go:462 +0x8c
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/packer/packer/rpc.(*Server).Serve(0xc820176600)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/packer/rpc/server.go:141 +0x274
2016/01/11 11:22:23 packer-builder-amazon-ebs: main.main()
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/plugin/builder-amazon-ebs/main.go:14 +0xc6
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 17 [syscall, locked to thread]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: runtime.goexit()
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 5 [syscall]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: os/signal.loop()
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by os/signal.init.1
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 18 [select, locked to thread]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: runtime.gopark(0x8ea920, 0xc82002e728, 0x7799d0, 0x6, 0x8ea818, 0x2)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/proc.go:185 +0x163
2016/01/11 11:22:23 packer-builder-amazon-ebs: runtime.selectgoImpl(0xc82002e728, 0x0, 0x18)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/select.go:392 +0xa64
2016/01/11 11:22:23 packer-builder-amazon-ebs: runtime.selectgo(0xc82002e728)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/select.go:212 +0x12
2016/01/11 11:22:23 packer-builder-amazon-ebs: runtime.ensureSigM.func1()
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/signal1_unix.go:227 +0x323
2016/01/11 11:22:23 packer-builder-amazon-ebs: runtime.goexit()
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 19 [chan receive]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/packer/packer/plugin.Server.func1(0xc82018a0c0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/packer/plugin/server.go:92 +0x66
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by github.com/mitchellh/packer/packer/plugin.Server
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/packer/plugin/server.go:96 +0xcb5
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 20 [IO wait]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.runtime_pollWait(0x19a0fa8, 0x72, 0xc82000a180)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/netpoll.go:157 +0x60
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*pollDesc).Wait(0xc82018c0d0, 0x72, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*pollDesc).WaitRead(0xc82018c0d0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*netFD).Read(0xc82018c070, 0xc820189000, 0x1000, 0x1000, 0x0, 0x1960050, 0xc82000a180)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/fd_unix.go:232 +0x23a
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*conn).Read(0xc820174048, 0xc820189000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/net.go:172 +0xe4
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).fill(0xc82018a240)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:97 +0x1e9
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).Read(0xc82018a240, 0xc8201ae000, 0xc, 0xc, 0xc8201ae002, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:207 +0x260
2016/01/11 11:22:23 packer-builder-amazon-ebs: io.ReadAtLeast(0x1c00000, 0xc82018a240, 0xc8201ae000, 0xc, 0xc, 0xc, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/io/io.go:298 +0xe6
2016/01/11 11:22:23 packer-builder-amazon-ebs: io.ReadFull(0x1c00000, 0xc82018a240, 0xc8201ae000, 0xc, 0xc, 0xc, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/io/io.go:316 +0x62
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Session).recvLoop(0xc820196000, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:398 +0x11e
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Session).recv(0xc820196000)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:386 +0x21
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by github.com/hashicorp/yamux.newSession
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:104 +0x4b1
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 21 [select]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Session).send(0xc820196000)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:348 +0x5e1
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by github.com/hashicorp/yamux.newSession
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:105 +0x4d3
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 22 [select]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Session).keepalive(0xc820196000)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:287 +0x107
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by github.com/hashicorp/yamux.newSession
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:107 +0x506
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 23 [select]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Session).AcceptStream(0xc820196000, 0x8ea048, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/session.go:191 +0x19d
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/mitchellh/packer/packer/rpc.(*muxBroker).Run(0xc8201765e0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/packer/rpc/mux_broker.go:126 +0x34
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by github.com/mitchellh/packer/packer/rpc.NewServer
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/mitchellh/packer/packer/rpc/server.go:42 +0x14a
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 81 [runnable]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/yamux.(*Stream).Read(0xc82026a000, 0xc820274000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/stream.go:125 +0x3ae
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).fill(0xc8202036e0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:97 +0x1e9
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).ReadByte(0xc8202036e0, 0xc8205a91f0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:229 +0x7a
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc820225bf0, 0x346ca1)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/decode.go:90 +0x42
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNext(0xc82025c880)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/msgpack.go:540 +0x41
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc820203740, 0x502cc0, 0xc820288060)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x70
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc820203740, 0x502cc0, 0xc820288060, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x74
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc8201cff40, 0x502cc0, 0xc820288060, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd2
2016/01/11 11:22:23 packer-builder-amazon-ebs: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeader(0xc8201cff40, 0xc820288060, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpack/codec/rpc.go:123 +0x54
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/rpc.(*Client).input(0xc8202037a0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/client.go:109 +0xbf
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by net/rpc.NewClientWithCodec
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/rpc/client.go:201 +0xd2
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 90 [IO wait]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.runtime_pollWait(0x19a1068, 0x72, 0xc82000a180)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/runtime/netpoll.go:157 +0x60
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*pollDesc).Wait(0xc82018c450, 0x72, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*pollDesc).WaitRead(0xc82018c450, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*netFD).Read(0xc82018c3f0, 0xc820581000, 0x800, 0x800, 0x0, 0x1960050, 0xc82000a180)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/fd_unix.go:232 +0x23a
2016/01/11 11:22:23 packer-builder-amazon-ebs: net.(*conn).Read(0xc820174078, 0xc820581000, 0x800, 0x800, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/net.go:172 +0xe4
2016/01/11 11:22:23 packer-builder-amazon-ebs: crypto/tls.(*block).readFromUntil(0xc820568000, 0x19a18f0, 0xc820174078, 0x5, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/crypto/tls/conn.go:455 +0xcc
2016/01/11 11:22:23 packer-builder-amazon-ebs: crypto/tls.(*Conn).readRecord(0xc82029c000, 0x8ea917, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/crypto/tls/conn.go:540 +0x2d1
2016/01/11 11:22:23 packer-builder-amazon-ebs: crypto/tls.(*Conn).Read(0xc82029c000, 0xc82029f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/crypto/tls/conn.go:901 +0x167
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/http.noteEOFReader.Read(0x1c46070, 0xc82029c000, 0xc8201961b8, 0xc82029f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/http/transport.go:1370 +0x67
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/http.(*noteEOFReader).Read(0xc8205be1e0, 0xc82029f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	<autogenerated>:126 +0xd0
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).fill(0xc820450360)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:97 +0x1e9
2016/01/11 11:22:23 packer-builder-amazon-ebs: bufio.(*Reader).Peek(0xc820450360, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/bufio/bufio.go:132 +0xcc
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/http.(*persistConn).readLoop(0xc820196160)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/http/transport.go:876 +0xf7
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by net/http.(*Transport).dialConn
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/http/transport.go:685 +0xc78
2016/01/11 11:22:23 packer-builder-amazon-ebs:
2016/01/11 11:22:23 packer-builder-amazon-ebs: goroutine 91 [select]:
2016/01/11 11:22:23 packer-builder-amazon-ebs: net/http.(*persistConn).writeLoop(0xc820196160)
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/http/transport.go:1009 +0x40c
2016/01/11 11:22:23 packer-builder-amazon-ebs: created by net/http.(*Transport).dialConn
2016/01/11 11:22:23 packer-builder-amazon-ebs: 	/usr/local/go/src/net/http/transport.go:686 +0xc9d
2016/01/11 11:22:23 ui error: Build 'amazon-ebs' errored: unexpected EOF
2016/01/11 11:22:23 Builds completed. Waiting on interrupt barrier...
2016/01/11 11:22:23 machine readable: error-count []string{"1"}
2016/01/11 11:22:23 ui error:
==> Some builds didn't complete successfully and had errors:
2016/01/11 11:22:23 machine readable: amazon-ebs,error []string{"unexpected EOF"}
2016/01/11 11:22:23 ui error: --> amazon-ebs: unexpected EOF
2016/01/11 11:22:23 ui:
==> Builds finished but no artifacts were created.
2016/01/11 11:22:23 waiting for all plugin processes to complete...
2016/01/11 11:22:23 /Users/*****/packer/packer-builder-amazon-ebs: plugin process exited



!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Packer crashed! This is always indicative of a bug within Packer.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Packer[1] so that we can fix this.

[1]: https://github.com/mitchellh/packer/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

It seems while not using a communicator it still tries to create a security group allowing access to a port that is not set as no ssh or winrm is used for the build.

**Comment by rickard-von-essen** [comment]
Similar to #2736

**Comment by tylert** [comment]
Related to #3304.

**Comment by rickard-von-essen** [comment]
This was fixed in 0.9.0 see #2848

**Comment by riponbanik** [comment]
came back on 1.0.2

**Comment by rickard-von-essen** [comment]
Hi @riponbanik, thanks for reaching out. 

This ticket is really old so it's probably not relevant to the issue you're seeing. If you're still having trouble, would you please open a new issue with the debug log and a json that reproduces the issue? Thanks!
